tests/core/pyspec/eth2spec/test/phase0/fork_choice/test_on_block.py

```diff
@@ -200,21 +203,31 @@ def test_on_block_finalized_skip_slots(spec, state):
     on_tick_and_append_step(spec, store, current_time, test_steps)
     assert store.time == current_time
 
-    # Create a finalized chain
-    for _ in range(4):
+    # Fill epoch 0 and the first slot of epoch 1
+    state, store, _ = yield from apply_next_slots_with_attestations(
+        spec, state, store, spec.SLOTS_PER_EPOCH, True, False, test_steps)
+
+    # Skip the rest slots of epoch 1 and the first slot of epoch 2
+    next_slots(spec, state, spec.SLOTS_PER_EPOCH)
+
+    # The state after the skipped slots
+    target_state = state.copy()
+
+    # Fill epoch 3 and 4
+    for _ in range(2):
         state, store, _ = yield from apply_next_epoch_with_attestations(
-            spec, state, store, True, False, test_steps=test_steps)
-    assert store.finalized_checkpoint.epoch == 2
+            spec, state, store, True, True, test_steps=test_steps)
 
-    # Another chain
-    another_state = store.block_states[store.finalized_checkpoint.root].copy()
-    # Build block that includes the skipped slots up to finality in chain
-    block = build_empty_block(spec,
-                              another_state,
-                              spec.compute_start_slot_at_epoch(store.finalized_checkpoint.epoch) + 2)
-    block.body.graffiti = b'\x12' * 32
-    signed_block = state_transition_and_sign_block(spec, another_state, block)
+    # Now we get finalized epoch 1, where compute_start_slot_at_epoch(1) is a skipped slot
```

Comment:
Is this comment correct?

Isn't it that `compute_start_slot_at_epoch(2)` is the skip slot?

Reply:
You're right. 👍 

```suggestion
    # Now we get finalized epoch 2, where `compute_start_slot_at_epoch(2)` is a skipped slot
```